Tests/SDWebImage Tests.xcodeproj/project.pbxproj

```diff
@@ -543,14 +561,14 @@
 		};
 		DA248D4A1954721A00390AB0 /* Debug */ = {
 			isa = XCBuildConfiguration;
-			baseConfigurationReference = EADD19EC219915E300804BB0 /* Module-Debug.xcconfig */;
```

Comment:
@bpoplauschi Seems previous the refactory of project in #2494, does not create a seperate xcconfig for the `SDWebImage Tests.xcodeproj`. But however, the test project and framework project have different configuration. So this time I create one to specify the different things.